### DISCUSSION TITLE
Restarting MD in LAMMPS

### DISCUSSION THREAD

**Discussion by bapatist** [ACEsuit/mace#196] [Q&A]
Hey all,
Is there restart support for continuing a LAMMPS-MD ? 
`write_restart   LAMMPS/restart.nvt`

I see the following warning and wanted to make sure whether its implemented or not
`WARNING: Pair style restartinfo set but has no restart support (src/pair.cpp:754)`

**Comment by pobo95** [comment]
Happens on the latest version of lammps. Try it before 2021 version
